ci/test.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
```

Comment:
We should not duplicate code here. What do you think about using a python module to share common functionality?